src/weekly_report.js

```diff
@@ -0,0 +1,65 @@
+const moment = require('moment')
+const today = moment()
+const todaysDate = moment()
+const minusSevenDays = todaysDate.subtract(7, 'days')
+console.log('-7: ', minusSevenDays.format("dddd, MMMM Do YYYY, h:mm:ss a"), '\n');
+console.log('todaysDate: ', today.format("dddd, MMMM Do YYYY, h:mm:ss a"));
+
+const lengthOfUse = (User, where) =>
+  User.findOne({where})
+    .then(user => {
+    return Math.abs(todaysDate.diff(user.createdAt, 'days'))
+    })
+
+const bulletsCreated = (auditRecords) => {
+  const createdThisWeek = auditRecords.filter(record => record.createdAt >= minusSevenDays)
+  return createdThisWeek.length
+  // ([all items w/ user_id && createdAt(<= today’s date - 7) ].length)
+  }
+
+const bulletsChanged = (auditRecords) => {
+  const changedThisWeek = auditRecords.filter(record => record.updatedAt >= minusSevenDays)
+  return changedThisWeek.length
+    // ([all items w/ user_id && updatedAt(<= today’s date - 7) ].length)
+}
+
+const totalBullets = (auditRecords) => {
+  const totalBullets = auditRecords.filter(record => {
+      return record.element_name === 'row' && record.createdAt >= minusSevenDays
```

Comment:
We should be able to construct a query to get this information, instead of doing this in the code.